src/main/java/homework_2/pyramid_printer/PyramidPrinterD.java

```diff
@@ -0,0 +1,27 @@
+package homework_2.pyramid_printer;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import static homework_2.pyramid_printer.Utils.*;
+
+public class PyramidPrinterD {
+    protected static void run() {
```

Comment:
You need to have a minimum of one method as public, otherwise, how to run/test it?